test/e2e/user_workloads_test.go

```diff
@@ -71,6 +72,11 @@ func (s *userWorkloadsTestSuite) TestIdlerAndPriorityClass() {
 		err := memberAwait.WaitUntilPodsDeleted(p.Namespace, wait.WithPodName(p.Name))
 		require.NoError(s.T(), err)
 	}
+	// check notification was created and Idler condition was updated
+	_, err = memberAwait.WaitForIdler("test-idler-dev", wait.IdlerConditions(Running(), IdledNotificationCreated()))
```

Comment:
Wouldn't there be a Notification created for the `test-idler-stage` namespace too?

Reply:
no, the short timeOut which makes the pod long running is only set for `test-idler-dev` namespace.

Reply:
Ah ok, that makes sense 👍 But in that case we should verify no notification was created for `test-idler-stage`

Reply:
done in [7ccb6b7](https://github.com/codeready-toolchain/toolchain-e2e/pull/560/commits/7ccb6b74640b9be103ebc902249245d76fcfc7d5)